src/client/common/errors/errorUtils.ts

```diff
@@ -41,7 +41,9 @@ export function getLastFrameFromPythonTraceback(
     }
     //             File "/Users/donjayamanne/miniconda3/envs/env3/lib/python3.7/site-packages/appnope/_nope.py", line 38, in C
 
-    const lastFrame = traceback
+    // This parameter might be either a string or a string array
+    const fixedTraceback: string = Array.isArray(traceback) ? traceback[0] : traceback;
```

Comment:
It makes more sense to change the parameter type, but the issue is that JS allows it to be either an array or a string. This works for both cases.